public/index.php

```diff
@@ -13,7 +13,7 @@
 
 DriverManager::loadDriver(SlackDriver::class);
 
-$config = parse_ini_file(__DIR__ . '/config.ini', true);
+$config = parse_ini_file(__DIR__ . '/../config.ini', true);
```

Comment:
I believe these changes will break dev environment until its docker-compose is updated.